x/stake/endblock.go

```diff
@@ -280,20 +281,33 @@ func handleRefundStake(ctx sdk.Context, sideChainPrefix []byte, k keeper.Keeper)
 	var refundEvents sdk.Events
 	count := 0
 	boundDenom := k.BondDenom(sideChainCtx)
+	bscSideChainId := k.ScKeeper.BscSideChainId(ctx)
+	unBoundingDelegationSize := k.GetUnbondingSizeInQueue(sideChainCtx, ctx.BlockHeader().Time.Add(k.UnbondingTime(ctx)))
```

Comment:
I can't get the point of adding these logic. Can you explain a bit?

Reply:
undelegation has been refactored, please help review